Dockerfile

```diff
@@ -72,8 +72,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
     imagemagick \
     groff \
     # For PPAs
-    software-properties-common
-
+    software-properties-common\
```

Comment:
space is missing after common and ther is no package like software-properties-commonjq